src/MongoDB.Driver.Core/Core/Operations/AggregateExplainOperation.cs

```diff
@@ -199,7 +199,10 @@ private ReadCommandOperation<BsonDocument> CreateOperation(SemanticVersion serve
                 _collectionNamespace.DatabaseNamespace,
                 command,
                 BsonDocumentSerializer.Instance,
-                _messageEncoderSettings);
+                _messageEncoderSettings)
+            {
+                RetryRequested = false
```

Comment:
Why we didn't add this parameter to the constructor? (there are a number of these cases)